app/helpers/status_helper.rb

```diff
@@ -6,14 +6,16 @@ module StatusHelper
     fundraising: 'Fundraising',
     pledge_sent: 'Pledge Sent',
     pledge_paid: 'Pledge Paid',
-    resolved: 'Resolved Without DCAF'
+    resolved: 'Resolved Without DCAF',
+    dropoff: 'Patient Has dropped Off'
   }.freeze
 
   def status
     return STATUSES[:resolved] if pregnancy.resolved_without_dcaf?
     return STATUSES[:pledge_sent] if pregnancy.pledge_sent?
     return STATUSES[:no_contact] if not contact_made?
     return STATUSES[:fundraising] if appointment_date
+    return STATUSES[:dropoff] if more_than_120
```

Comment:
Can you swap `more_than_120` out here for `duration_since_last_call > 120`, and move this to the top of the `status` method? We want to check for this first in the sequence.